homeassistant/components/homekit_controller/sensor.py

```diff
@@ -524,14 +530,53 @@ def native_value(self) -> str | int | float:
 }
 
 
+class RSSISensor(HomeKitEntity, SensorEntity):
+    """HomeKit Controller RSSI sensor."""
+
+    _attr_device_class = SensorDeviceClass.SIGNAL_STRENGTH
+    _attr_entity_category = EntityCategory.DIAGNOSTIC
+    _attr_entity_registry_enabled_default = False
+    _attr_has_entity_name = True
+    _attr_native_unit_of_measurement = SIGNAL_STRENGTH_DECIBELS_MILLIWATT
+    _attr_should_poll = False
+
+    def get_characteristic_types(self) -> list[str]:
+        """Define the homekit characteristics the entity cares about."""
+        return []
+
+    @property
+    def available(self) -> bool:
+        """Return if the bluetooth device is available."""
+        address = self._accessory.pairing_data["AccessoryAddress"]
+        return async_ble_device_from_address(self.hass, address) is not None
+
+    @property
+    def name(self) -> str:
+        """Return the name of the sensor."""
+        return "Signal strength"
+
+    @property
+    def unique_id(self) -> str:
+        """Return the ID of this device."""
+        serial = self.accessory_info.value(CharacteristicsTypes.SERIAL_NUMBER)
+        return f"homekit-{serial}-rssi"
```

Comment:
no need for `homekit` since it's checked for uniqueness combined with `homekit_controller` + `sensor`. Unless you're trying to be conform the other unique IDs 👍

Reply:
I prefixed it only for consistency since it will make it easier when we fix https://github.com/home-assistant/core/issues/60667